src/test/java/org/prebid/server/bidder/outbrain/OutbrainBidderTest.java

```diff
@@ -225,6 +229,55 @@ public void makeBidsShouldReturnResponseWithErrorWhenIdIsNotFound() throws JsonP
         assertThat(result.getValue()).isEmpty();
     }
 
+    @Test
+    public void makeBidsShouldReturnNativeBidWithModifiedImpIfNativeIsPresent() throws JsonProcessingException {
+        // given
+        final List<EventTracker> eventTrackers = new ArrayList<>();
```

Comment:
can be created with Arrays.asList method, and used without separate variable creation